<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgFunkalphabet</class>
 <widget class="QDialog" name="DlgFunkalphabet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>845</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Funkalphabet</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QGroupBox" name="gNorm">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Legt die Kodierung fest.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="title">
        <string>Kodierungsnorm</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QRadioButton" name="rkDIN">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Kodierung gemäß DIN.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>DIN</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rkITU">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Kodierung gemäß ITU.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>ITU</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="sfDrucken">
     <property name="toolTip">
      <string>Druckt beide Normen als Handzettel aus.</string>
     </property>
     <property name="text">
      <string>Drucken</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="Karten">
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="karteUebersicht">
      <attribute name="title">
       <string>Übersicht</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTableView" name="tbUebersicht">
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAsNeeded</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="dragDropOverwriteMode">
          <bool>false</bool>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
         <property name="horizontalScrollMode">
          <enum>QAbstractItemView::ScrollPerItem</enum>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="karteLiveuebersetzung">
      <attribute name="title">
       <string>Übersetzung</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QLineEdit" name="txtEingabe">
          <property name="toolTip">
           <string>Bitte den Text eingeben.</string>
          </property>
         </widget>
         <widget class="QTextBrowser" name="txtAusgabe">
          <property name="acceptDrops">
           <bool>false</bool>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="bbEnde">
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>rkDIN</tabstop>
  <tabstop>rkITU</tabstop>
  <tabstop>Karten</tabstop>
  <tabstop>tbUebersicht</tabstop>
  <tabstop>sfDrucken</tabstop>
  <tabstop>txtEingabe</tabstop>
  <tabstop>txtAusgabe</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>bbEnde</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>DlgFunkalphabet</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>788</x>
     <y>521</y>
    </hint>
    <hint type="destinationlabel">
     <x>831</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
